Gruntfile.js

```diff
@@ -49,8 +46,9 @@ module.exports = function(grunt) {
     babel: {
       dist: {
         files: [{
+          cwd: 'src/js',
           expand: true,
-          src: ['src/js/*.js'],
+          src: ['*.js'],
```

Comment:
Using `src: [src/js/*.js]` will create `es5/src/js/*.js` which is not really necessary for this project at the moment because it is just one file.